feature/account/setup/src/main/res/values/strings.xml

```diff
@@ -11,18 +11,23 @@
 
     <string name="account_setup_outgoing_config_top_bar_title">Outgoing server settings</string>
     <string name="account_setup_outgoing_config_server_label">Server</string>
+    <string name="account_setup_outgoing_config_server_error_required">Server name is required.</string>
     <string name="account_setup_outgoing_config_security_label">Security</string>
     <string name="account_setup_outgoing_config_security_none">None</string>
     <string name="account_setup_outgoing_config_security_ssl">SSL/TLS</string>
     <string name="account_setup_outgoing_config_security_start_tls">StartTLS</string>
     <string name="account_setup_outgoing_config_port_label">Port</string>
+    <string name="account_setup_outgoing_config_port_error_required">Port is required.</string>
+    <string name="account_setup_outgoing_config_port_error_invalid">Port is invalid (0–65535).</string>
```

Comment:
I think we should make it clearer that the given range is valid values, not invalid values.

```suggestion
    <string name="account_setup_outgoing_config_port_error_invalid">Port is invalid (must be 1–65535).</string>
```